src/main/java/org/hashids/Hashids.java

```diff
@@ -10,379 +10,370 @@
  * database IDs, use them as forgotten password hashes, invitation codes, store shard numbers.
  * <p>
  * This is implementation of http://hashids.org v1.0.0 version.
- *
+ * <p>
  * This implementation is immutable, thread-safe, no lock is necessary.
  *
  * @author <a href="mailto:fanweixiao@gmail.com">fanweixiao</a>
  * @author <a href="mailto:terciofilho@gmail.com">Tercio Gaudencio Filho</a>
  * @since 0.3.3
  */
 public class Hashids {
-  /**
-   * Max number that can be encoded with Hashids.
-   */
-  public static final long MAX_NUMBER = 9007199254740992L;
-
-  private static final String DEFAULT_ALPHABET = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";
-  private static final String DEFAULT_SEPS = "cfhistuCFHISTU";
-  private static final String DEFAULT_SALT = "";
-
-  private static final int DEFAULT_MIN_HASH_LENGTH = 0;
-  private static final int MIN_ALPHABET_LENGTH = 16;
-  private static final double SEP_DIV = 3.5;
-  private static final int GUARD_DIV = 12;
-
-  private final String salt;
-  private final int minHashLength;
-  private final String alphabet;
-  private final String seps;
-  private final String guards;
-
-  public Hashids() {
-    this(DEFAULT_SALT);
-  }
-
-  public Hashids(String salt) {
-    this(salt, 0);
-  }
-
-  public Hashids(String salt, int minHashLength) {
-    this(salt, minHashLength, DEFAULT_ALPHABET);
-  }
-
-  public Hashids(String salt, int minHashLength, String alphabet) {
-    this.salt = salt != null ? salt : DEFAULT_SALT;
-    this.minHashLength = minHashLength > 0 ? minHashLength : DEFAULT_MIN_HASH_LENGTH;
-
-    final StringBuilder uniqueAlphabet = new StringBuilder();
-    for (int i = 0; i < alphabet.length(); i++) {
-      if (uniqueAlphabet.indexOf(String.valueOf(alphabet.charAt(i))) == -1) {
-        uniqueAlphabet.append(alphabet.charAt(i));
-      }
+    /**
```

Comment:
Looks like your IDE is indenting using 4 spaces instead of 2. The fix will make the review to identify the changes.